app/models/service_template_ansible_playbook.rb

```diff
@@ -89,7 +89,7 @@ def self.build_parameter_list(name, description, info)
       :description              => description || '',
       :project                  => playbook.configuration_script_source.manager_ref,
       :playbook                 => playbook.name,
-      :inventory                => tower.inventory_root_groups.first.ems_ref,
+      :inventory                => tower.inventory_root_groups.find_by!(:name => 'Demo Inventory').ems_ref,
```

Comment:
Not sure where should the constant be replaced. We can refactor once the solution is finalized.